src/createEpicMiddleware.js

```diff
@@ -45,6 +48,7 @@ export function createEpicMiddleware(epic, options = defaultOptions) {
           return output$;
         })
         ::switchMap(output$ => options.adapter.output(output$))
+        ::mergeMap(action => of(action, asap))
```

Comment:
I tried using `::observeOn(asap)` first, but it broke the `it('should throw if you provide a root epic that doesn\'t return anything)'` since exception was thrown from the `asap` scheduler and not from `createStore(..., applyMiddleware(...))` call.